src/main/resources/application.properties

```diff
@@ -49,6 +49,7 @@ scheduler.executors.thread.pool.size=30
 scheduler.jobs.parallel.number=100
 scheduler.heart.beat=5000
 scheduler.lag.threshold=20000
+scheduler.sensors.changedSensorsMinimalChunkQuerySize=100
```

Comment:
I'd have been ok with the value hard-coded in the configs, just not in the service itself, but anyway, now you've already done it.
It seems that we don't use camelCase in the properties files. Almost all properties are lower case with a dot between words. The only exception is javamelody which uses kebab case

Reply:
Taking this back. camelCase is still better than dots, since the dots imply a hierarchy

Reply:
Ok